modules/ROOT/pages/manage-backup-restore.adoc

```diff
@@ -20,23 +20,32 @@ Individual node backup and restore operations are not supported.
 
 == Create a Backup
 
-To create a backup, run the following command on any controller node in the cluster as a privileged user (with root access):
+To create a backup:
```

Comment:
I would use the Step 1 text as the intro sentence. Something like: 

To create a backup, run the backup command on any controller node in the Runtime Fabric cluster, as a privileged user with root access. 
Then step one could be:
Run the following command:

Suggestion only.